src/main/kotlin/com/sourcegraph/cody/edit/FixupService.kt

```diff
@@ -45,13 +75,20 @@ class FixupService(val project: Project) : Disposable {
     TestCodeSession(this, editor, editor.project ?: return)
   }
 
+  @RequiresEdt
   fun isEligibleForInlineEdit(editor: Editor): Boolean {
     if (!isCodyEnabled()) {
       logger.warn("Edit code invoked when Cody not enabled")
       return false
     }
     if (!CodyEditorUtil.isEditorValidForAutocomplete(editor)) {
-      logger.warn("Inline edit invoked when editing not available")
+      logger.warn("Edit code invoked when editing not available")
+      return false
+    }
+    val policy = IgnoreOracle.getInstance(project).policyForEditor(editor)
+    if (policy != IgnorePolicy.USE) {
+      runInEdt { ActionInIgnoredFileNotification().notify(project) }
```

Comment:
If this requires EDT, then it isn't worth spinning the event loop to show the notification, just show it.